arwen/contexts/storage.go

```diff
@@ -21,36 +20,32 @@ type storageContext struct {
 	stateStack                    [][]byte
 	elrondProtectedKeyPrefix      []byte
 	arwenStorageProtectionEnabled bool
-
-	useDifferentGasCostForReadingCachedStorageEpoch uint32
-	flagUseDifferentGasCostForReadingCachedStorage  atomic.Flag
+	enableEpochsHandler           vmcommon.EnableEpochsHandler
```

Comment:
Because the `EnableEpochsHandler` is already on the `host`, it should not be a field of the `StorageContext`. The `RuntimeContext` already uses the `EnableEpochsHandler` from the `host` directly.

Reply:
pushed